package.json

```diff
@@ -104,19 +103,18 @@
   },
   "scripts": {
     "prebuild": "node bin/fetch_api_docs_and_roadie_local.js",
-    "build": "yarn clean && yarn prebuild && gatsby build",
+    "build": "yarn prebuild && gatsby build",
     "tech-docs:build": "bash bin/convert_tech_docs.sh",
     "tech-docs:clean": "rm -rf roadie-tech-docs",
     "develop": "gatsby develop",
     "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
-    "start": "gatsby develop",
-    "start-with-env": "env $(cat .env | xargs) yarn start",
+    "start": "netlify dev",
```

Comment:
Using the netlify dev server instead of bare gatsby provides a local implementation of netlify functions and the netlify blob store. The site depends on these now so it's good to have them available during development.